### PR TITLE
Insurance payment screen

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CardholdersNameInput.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+type Props = {| +onCardholdersNameChange: () => void |};
+
+export default function CardholdersNameInput(props: Props) {
+  return (
+    <React.Fragment>
+      <Text style={styles.header}>
+        <Translation id="mmb.trip_services.insurance.payment.cardholders_name" />
+      </Text>
+      <TextInput defaultValue="" onChangeText={props.onCardholdersNameChange} />
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CardholdersNameInput.js]: I don't think this defaultvalue="" is necessary

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CreditCardNumberInput.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Props = {| +onCreditCardChange: () => void |};
+
+export default function CreditCardNumberInput(props: Props) {
+  return (
+    <React.Fragment>
+      <Text style={styles.header}>
+        <Translation id="mmb.trip_services.insurance.payment.credit_card_number" />
+      </Text>
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CreditCardNumberInput.js]: Maybe we should add a label prop to `TextInput`

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CreditCardNumberInput.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      </View>
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  header: { color: defaultTokens.colorTextSecondary },
+  image: {
+    position: 'absolute',
+    alignSelf: 'flex-end',
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CreditCardNumberInput.js]: it works, but maybe `end: 0,` is better with `position: 'absolute'`?

**Comment by RobinCsl** [comment]
@tbergq 🚀

**Pull request merged**